### ISSUE TITLE
Fix and re-enable `ia-components` tests

### ISSUE THREAD

**Issue by jbuckner** [internetarchive/iaux#550]
**Task Details**

> The ia-components tests are timing out with some errors so we disabled them in https://github.com/internetarchive/iaux/pull/549. Fix and re-enable the tests.

**Verification**

> ia-components tests should pass during CI

**Stakeholders**

> @iisa @cdrini

**Comment by crew-guy** [comment]
Hey ! I would like to work on this. How can I proceed about this ?

**Comment by sarthakkundra** [comment]
Hey @crew-guy I think you should run `yarn test` inside the said directory which will tell you the failing tests. You can then proceed about to fixing them :)

**Comment by crew-guy** [comment]
Gotcha ! Thanks. I'm working on this !

**Comment by sarthakkundra** [comment]
@crew-guy Are you still working on this?

**Comment by crew-guy** [comment]
Yeah ! I was actually down with covid for a while so I was resting but I have resumed programming since the past 3 days. I am working on this issue right now

**Comment by sarthakkundra** [comment]
Hey @crew-guy just checking in. Any progress / blockers?

**Comment by crew-guy** [comment]
Sorry. I got busy with a couple of my friends to help verify leads for covid and build a portal for the same. I know what I went through so I started helping other people with the same. I'll resume working once I reach back home after 5 hours

**Comment by soham4abc** [comment]
I am working on this issue right away!
